.github/CONTRIBUTING.md

```diff
@@ -9,6 +9,39 @@ All assets including icons and sound are under a [Creative Commons 3.0 BY-SA](ht
 
 # Github Standards
 
+### Sub-licensing External Content
+**When does this section apply to me?** When you are integrating content that is **not** licensed under [AGPLv3](https://www.gnu.org/licenses/agpl-3.0.en.html) (code)
+or [Creative Commons 3.0 BY-SA](https://creativecommons.org/licenses/by-sa/3.0/) (icons and sounds). The most common application here is for icons and sounds gathered
+from an external source or repository.
+
+As a contributor, you must do your utmost to pay respect to international copyright law. This means that, if copying code or content from an external source, you **must**
+be aware of what license it is published under. And you must ensure that the conditions of said license are followed when integrating the content into the codebase. (Sometimes
+this is not possible, and the content cannot be used.)
+
+**If you cannot locate or reasonably dicern authorship or the license associated with given content, we ask that you not submit it into a pull request.**
+
+As a courtesy, attribution of authorship is also encouraged, even if the license of the content does not require this outright. Generally this is done in the PR by declaring
+the source of the material, the author(s) (if known), and adding similar attributions to the relevant changelog entry (if applicable). This may be both done in the `author` field
+or within the plain text describing the change itself.
+
+When reading the sub-sections that follow, note that these will be **generalizations**, and the specifics will be dictated by the license itself.
+
+#### Code
+Code from most other open source SS13 code bases is GPLv3 or AGPLv3. This means that it's free to be copied without any additional effort. Though any notes of authorship within
+the code files, if presents, must be kept intact. Be wary of **Goon**: their code is licensed under CC-BY-SA-NC and is **not** directly compatible with our codebase's license.
+Porting Goon code directly is highly discouraged as a result.
+
+If the code is external, or licensed under something else (example being the TGS library), then ensure that the copyright notices within the file(s) (if present) are kept intact,
+and that a separate license file (if present in/packaged with the original source) is added to the repository. This generally means that you have to put all of the code you are
```

Comment:
Wouldn't it be simpler to include a link to the license in code comments? Especially when there's ported and own code mixed up in the same file?

Reply:
Not how it works. Licenses usually directly state that the license notice in question CANNOT be removed. This means that:
* If it's written into the code file, it must be within the code file.
* If it's in a folder, you must transfer the whole license.

You also cannot remove the authorship attribute usually.

But you can add a note of, "Modified by" blablabla.

Reply:
A link to the license notice doesn't count?